src/query.ts

```diff
@@ -37,12 +36,16 @@ const pageLimit = 100;
  * @example
  * const query = buildQuery([{ name: 'Folder-Id', value: folderId }]);
  */
-export function buildQuery(tags: GQLTagInterface[], cursor?: string, owner?: ArweaveAddress): GQLQuery {
+export function buildQuery(
+	tags: { name: string; value: string | string[] }[],
```

Comment:
Write a TODO here to update the GQLTagInterface type when we move back into Core.

Reply:
(and maybe a tech debt ticket)

Reply:
I created a ticket: https://ardrive.atlassian.net/browse/PE-465